fe/fe-core/src/main/java/com/starrocks/analysis/CreateTableStmt.java

```diff
@@ -330,7 +332,7 @@ public void analyze(Analyzer analyzer) throws UserException {
                 }
             }
         } else {
-            // mysql, broker and hive do not need key desc
+            // mysql, broker, iceberg and hive do not need key desc
```

Comment:
The keydesc here is used for the data model in starrocks like 'uniq key' 'primary key' 'agg key' 'duplicate key'.
For external table we do not support key desc since we just support query. We will not write data into iceberg like fink sink currently. @openinx we will consider this scenario when we support write data into iceberg table in starrocks